src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-apm-ui.mdx

```diff
@@ -0,0 +1,171 @@
+- `rpc.grpc.status_code` from [rpc span][] semantic conventions
+- `http.status_code` from [http span][] semantic conventions
+- `undefined` if none of the above are present
+
+## Page: Metrics explorer [#metrics-explorer-page]
+
+The Metrics explorer provides tools for exploring a service's metrics in a generic manner. See [explore your data](https://docs.newrelic.com/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer/#explore-data) for more information.
+
+## Golden signals [#golden-signals]
+
+The golden signals of throughput, response time, and error rate appear in several places throughout the OpenTelemetry APM UI. When used, they are computed as follows:
+
+For **metrics**, the queries assume data conforms to the [http metric][] or [rpc metric][] semantic conventions.
+
+For **spans**, the queries are generic, utilizing only the top level span data model. Spans are counted towards throughput and response time if they are root entry spans into a service, computed using a heuristic of `WHERE span.kind = server OR span.kind = consumer`. Spans are errors if they have a status code of `ERROR` (i.e. `otel.status_code = ERROR`).
+
+## Narrow data with filters [#narrow-with-filters]
+
+Several pages include a filter bar (i.e. **Narrow data to..**). This allows you to filter queries on the page to match the criteria. For example, you might narrow to a particular canary deployment by filtering for `service.version='1.2.3-canary'`. Filters are preserved when navigating between pages.
+
+## Metrics or spans toggle [#metrics-spans-toggle]
+
+Various pages include a span-metrics toggle. This allows you to toggle whether queries are driven by span or metric data based on analysis requirements and data availability.
+
+Metrics are not subject to sampling, and are thus more accurate, especially when computing rates like throughput. However, metrics are subject to cardinality constraints, and may lack certain attributes important for analysis. In contrast, spans are sampled and thus subject to accuracy issues, but have richer attributes since they are not subject to cardinality constraints.
+
+Historically, OpenTelemetry language APIs / SDKs and instrumentation prioritized trace instrumentation. However, the project has come a long way and [metrics](https://opentelemetry.io/docs/languages/) are available in almost all languages. Check the documentation of the relevant language and instrumentation for more details.
+
+[http metric]: https://opentelemetry.io/docs/specs/semconv/http/http-metrics/
+[http span]: https://opentelemetry.io/docs/specs/semconv/http/http-spans/
+[rpc metric]: https://opentelemetry.io/docs/specs/semconv/rpc/rpc-metrics/
+[rpc span]: https://opentelemetry.io/docs/specs/semconv/rpc/rpc-spans/
+[service entity attributes]: /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources/#services
```

Comment:
@jack-berg What's the purpose of these doc links? They should have some kind of preface beforehand, or be connected to the paragraph above

Reply:
These do not get rendered. It's a feature of markdown that enables you to create a reference to links that are used multiple times within the document.